Main.java

```diff
@@ -9,7 +10,39 @@
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
+    // Determines if number is a Fizz, Buzz, or FizzBuzz
+
+    // Create a scanner for user input
+    Scanner input = new Scanner(System.in);
+    
+    // prompt the user for a number
+    System.out.print("Please enter a number:");
+
+    // get the number from the user
+    int number = input.nextInt();
+
+    // determine weather the number is a Fizz
+    int fizz = 0;
+
```

Comment:
I don't know if this is a good name for this. It is representing the "is divisible by" check. It should really be a constant as well.